### PR TITLE
New API Overview - Publisher

### PR DESCRIPTION
**Pull request by chanaka3d**
## Purpose

New Overview page and Endpoint view
ui

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+                                <Grid
+                                    container
+                                    direction='row'
+                                    alignItems='flex-start'
+                                    spacing={16}
+                                    className={classes.buttonSection}
+                                >
+                                    <Grid item>
+                                        <Button type='submit' variant='contained' color='primary'>
+                                            Save
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx]: We need a formatted message here

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+                                                <FormattedMessage
+                                                    id='api.create.endpoint.help'
+                                                    defaultMessage='This is the actual endpoint where the API implementation can be found'
+                                                />
+                                            )
+                                        }
+                                        InputLabelProps={{
+                                            shrink: true,
+                                        }}
+                                        label='Sandbox Endpoint'
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx]: Missing Formatted message

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                                                <FormattedMessage
+                                                    id='api.create.endpoint.help'
+                                                    defaultMessage='This is the actual endpoint where the API implementation can be found'
+                                                />
+                                            )
+                                        }
+                                        InputLabelProps={{
+                                            shrink: true,
+                                        }}
+                                        label='Production Endpoint'
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx]: Missing Formatted message

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx [hunk 4]
```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Endpoints/Endpoints.jsx]: license year is old

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/.eslintignore [hunk 5]
```diff
@@ -6,4 +6,5 @@
 /source/src/app/components/Apis/Details/LifeCycle/**
 /source/src/app/components/Apis/Details/Documents/**
 /source/src/app/components/Apis/Details/Endpoints/**
+/source/src/app/components/Apis/Details/NewOverview/**
```
Comment (malinthaprasan) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/.eslintignore]: How about renaming the old Overview as OldOverview and keep the new one as "Overview" itself. Then we can delete the OldOverview files easily later without the need to change the code.

**Comment by dushansilva** [comment]
https://github.com/wso2/product-apim/issues/4731

**Pull request merged**